rpc/gnmi/gnmi-specification.md

```diff
@@ -122,7 +122,7 @@ Timestamp values MUST be represented as the number of nanoseconds since the Unix
 
 ### 2.2.2 Paths
 
-Paths are represented according to [gNMI Path Conventions](https://github.com/openconfig/reference/blob/master/rpc/gnmi/gnmi-path-conventions.md), a simplified form of XPATH. Rather than utilising a single string to represent the path - with the "/" character separating each element of the path, the path is represented by an ordered list of strings, starting at the root node, and ending at the most specific path element.
+Paths are represented according to [gNMI Path Conventions](https://github.com/openconfig/reference/blob/master/rpc/gnmi/gnmi-path-conventions.md), a simplified form of XPATH. Rather than utilising a single string to represent the path - with the `/` character separating each element of the path, the path is represented by an ordered list of strings, starting at the root node, and ending at the most specific path element.
 
```

Comment:
I didn't change any of these because the doc is pretty inconsistent as far as British English vs American English, and actually the majority seems to be on the British English side.  I can send another PR if you guys are OK switching everything over to American English.